MultiServer.py

```diff
@@ -1093,6 +1093,20 @@ def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bo
                                        f"did you mean '{picks[0][0]}'? ({picks[0][1]}% sure)"
 
 
+def get_input_text_from_response(text: str, command: str) -> typing.Optional[str]:
```

Comment:
I like the choice to define `get_input_text_from_response` right below `get_intended_text`, so that they are together.

I think that making them both static methods of one class would be even more effective at keeping them together.
(I could see someone in the future sticking other unrelated stuff between them.)

But it seems like `MultiServer.py` might not be the best place for them.
Maybe they could both be moved to `Utils.py`